frontend/src/js/controllers/challengeCreate.js

```diff
@@ -9,7 +9,7 @@
 
     ChallengeCreateCtrl.$inject = ['utilities', '$state', '$http', '$rootScope'];
```

Comment:
No need to inject `$http` and `$rootScope` since these are not used anywhere. Please remove it.